### DISCUSSION TITLE
bends and volumeand balance

### DISCUSSION THREAD

**Discussion by blindfaced** [helge17/tuxguitar#61] [General]
hi

i noticed that full tone bends are labelled f  ( that means forte in music notation ) the correct label would be 1;
what about replacing the instrument column by a volume and balance settings to save time ( in early days there was a window called matrix or something like that ) but you had to open it via the menu bar. in my opinion that would be great to have something like in another well known guitar soft

cheers

**Comment by guiv42** [comment]
Thank you for the feedback on bend representation. The most widespread label I found for a full tone bend was "full", but it was a bit too wide and could easily create conflicts with other displayed items. I did not think of the possible confusion between "f" and "forte". No problem to replace "f" by "1".
About volume and balance, I don't really understand your proposal. Are you referring to the "Instruments" dialog? (see screenshot)
![Instruments](https://github.com/helge17/tuxguitar/assets/129443524/e2ffbe82-7cf5-44c2-9963-54c2606fc2f8)
